lib/rubocop/cop/rspec/rails/have_http_status.rb

```diff
@@ -20,6 +20,8 @@ class HaveHttpStatus < Base
             'Prefer `expect(response).%<to>s have_http_status(%<status>i)` ' \
             'over `expect(response.status).%<to>s %<match>s`.'
 
+          RESTRICT_ON_SEND = Runners.to_a
```

Comment:
`.to_a` looks weird, as `RESTRICT_ON_SEND` can be assigned a `Set`.

I don't have an idea for a better name.
`ALL` is a private API, implementation detail. And using `ALL` would make it inconsistent with other cops that would use non-static DSL configuraton  e.g. `Language.config['Expectations']` as you say above.

Reply:
This may not be the best, but how about `configurations`?

Reply:
Maybe `each`?

It's possible to double purpose `.all`:
```ruby
def self.all(element = NO_ARGUMENTS)
  return ALL if element == NO_ARGUMENTS
  ALL.include?(element)
end
```

I have no strong preference.